template_files/template.yml

```diff
@@ -0,0 +1,24 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: >
+  Sample Julia SAM Template
+
+Resources:
+  JuliaFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      PackageType: Image
+      MemorySize: 1028
+      Timeout: 60
+    Metadata:
+      Dockerfile: Dockerfile
+      DockerContext: ./template_files
+      DockerTag: julia1.6-v1
+
+Outputs:
+  JuliaFunctionLambdaArn:
+    Description: "Julia Lambda Function ARN"
+    Value: !GetAtt JuliaFunction.Arn
+  JuliaFunctionRoleArn:
+    Description: "Implicit IAM Role created for the Julia function"
+    Value: !GetAtt JuliaFunctionRole.Arn
```

Comment:
Where is this defined?

Reply:
This isn't well documented, but this is implicitly created by AWS SAM.

- https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-getting-started-hello-world.html

- https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-resource-function.html

Reply:
Maybe add one of the links to the description so people know where to look for details.